CryptoPrices/Sources/MyCoin/Sources/MyCoin/CoinStatisticsSection/CoinStatisticsSection.swift

```diff
@@ -4,6 +4,7 @@
 //
 //  Created by Khanh on 03/01/2023.
 //
+// swiftlint:disable void_function_in_ternary
```

Comment:
Can we fix this by turning the `isPriceUp` ternary check into an `if else` check? I think we should try to fix the raised swiftlint issues rather than ignoring them unless there is no way to resolve them with our convention 🙏

Reply:
Thank you so much 🙏. I fixed it at https://github.com/nimblehq/nimble-crypto-ios/pull/96/commits/a000bbe862a1a053b222617697dfc41585b1a87c.